google/resumable_media/_helpers.py

```diff
@@ -187,7 +189,9 @@ def wait_and_retry(func, get_status_code, retry_strategy):
 
             return response
 
-        base_wait, wait_time = calculate_retry_wait(base_wait, retry_strategy.max_sleep)
+        base_wait, wait_time = calculate_retry_wait(
+            base_wait, retry_strategy.max_sleep, retry_strategy.multiplier
+        )
```

Comment:
Kind of surprising that this is all calculated in the library rather than using a helper (in Go we use gax) but obviously nothing to address here.

Reply:
Strong agree. We have such a helper for python-storage but because the transport layer is different I couldn't simply replace the implementation here with that helper. In the future I'd like to do a bit more work to bring resumable media into the fold in terms of unifying its behavior with python-storage and BQ.